src/services/dashboards/dashboard-customize/modules/dashboard-widget-input-form/DashboardWidgetInputForm.vue

```diff
@@ -245,6 +248,12 @@ export default defineComponent<Props>({
             // update form data
             state.schemaFormData = { ...state.schemaFormData, [propertyName]: undefined };
         };
+        const disableInheritToggle = (propertyName: string) => {
```

Comment:
nitpick: I think the name of this function is ambiguous. What about `isInheritToggleDisabled` or `getInheritToggleDisabledState` ?

Reply:
suggestion: I think the logics for project dashboard case are distributed.
What about to use `widgetOptionsJsonSchema`?